.github/workflows/dart.yml

```diff
@@ -12,18 +12,19 @@ jobs:
 
     steps:
       - uses: actions/checkout@v3
+      - uses: subosito/flutter-action@v2
 
       # Note: This workflow uses the latest stable version of the Dart SDK.
       # You can specify other versions if desired, see documentation here:
       # https://github.com/dart-lang/setup-dart/blob/main/README.md
       - uses: dart-lang/setup-dart@v1
 
       - name: Install dependencies
-        run: dart pub get
+        run: flutter pub get
 
       # Uncomment this step to verify the use of 'dart format' on each commit.
       - name: Verify formatting
-        run: dart format --output=none --set-exit-if-changed .
+        run: dart format --output=none --set-exit-if-changed . -l 120
```

Comment:
is it possible to have the -l param in a config file? so that it's possible to just run `dart format` in the terminal and get the same result

Reply:
nope https://github.com/dart-lang/dart_style/wiki/FAQ#why-cant-i-configure-it

Reply:
We can of course go with the default 80 char limit